### PR TITLE
Decision transformers tutorial

### PR DESCRIPTION
**Pull request by simoninithomas**
Hey there 👋 ,

We have finished the tutorial on how to train your Decision Transformer from scratch.
It's already reviewed. We just need to add the colab link.

Edit: the article will be published tuesday.

### CODE REVIEW

File: _blog.yml [hunk 1]
```diff
@@ -1196,3 +1196,11 @@
   date: August 31, 2022
   tags:
     - community
+ 
+- local: decision-transformers-train
```
Comment (osanseviero) [inline _blog.yml:1200]: Nit: WDYT Of "train-decision-transformers" instead?

File: _blog.yml [hunk 2]
```diff
@@ -1196,3 +1196,11 @@
...
+  title: "Train your first Decision Transformer"
+  author: edbeeching
+  thumbnail: /blog/assets/101_decision-transformers-train/thumbnail.gif
+  date: September 06, 2022
```
Comment (osanseviero) [inline _blog.yml:1204]: To update

File: decision-transformers-train.md [hunk 3]
```diff
@@ -0,0 +1,298 @@
+---
+title: "Train your first Decision Transformer"
+thumbnail: /blog/assets/101_decision-transformers-train/thumbnail.gif
+---
+
+# Train your first Decision Transformer
+
+<div class="blog-metadata">
+    <small>Published September 06, 2022.</small>
```
Comment (osanseviero) [inline decision-transformers-train.md:9]: To update

**Review by osanseviero — COMMENTED** [review]
Thanks for this!

File: _blog.yml (hunk 1 above)
Comment (simoninithomas) [reply _blog.yml:1200]: Ok I modify it ✔️

File: _blog.yml (hunk 2 above)
Comment (simoninithomas) [reply _blog.yml]: ✅

File: decision-transformers-train.md (hunk 3 above)
Comment (simoninithomas) [reply decision-transformers-train.md]: ✅

**Comment by simoninithomas** [comment]
Thanks @osanseviero for the review 🤗

File: train-decision-transformers.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ thumbnail: /blog/assets/101_train-decision-transformers/thumbnail.gif
 
 In a [previous post](https://huggingface.co/blog/decision-transformers), we announced the launch of Decision Transformers in the transformers library. This new technique of **using a Transformer as a Decision-making model** is getting increasingly popular.
 
-So today, **you’ll learn to train your first Offline Decision Transformer model from scratch to make a half-cheetah run.** We'll train it directly on a Google Colab that you can find here. [ADD THE COLAB LINK]
+So today, **you’ll learn to train your first Offline Decision Transformer model from scratch to make a half-cheetah run.** We'll train it directly on a Google Colab that you can find here 👉 https://github.com/huggingface/blog/blob/main/notebooks/101_train-decision-transformers.ipynb
```
Comment (osanseviero) [inline train-decision-transformers.md:34]: You could also have a "Open in Colab" as done in https://huggingface.co/blog/stable_diffusion

**Pull request merged**